### PR TITLE
Update the BatchDTO as per the api spec

### PR DESCRIPTION
**Pull request by danishjamal104**
Updates in this PR
1. Updated `BatchDTO.java` to parse the batch-summary API from operations-app.
2. Updated the logic to support result CSV file.

### CODE REVIEW

**Comment by danishjamal104** [comment]
@avikganguly01 and @fynmanoj please review and merge. Thanks :)

File: src/main/java/org/mifos/processor/bulk/camel/routes/BatchStatusRoute.java [hunk 1]
```diff
@@ -65,7 +65,7 @@ public void configure() throws Exception {
                 .process(exchange -> {
                     BatchDTO batchSummary = exchange.getIn().getBody(BatchDTO.class);
 
-                    long percentage = (batchSummary.getCompleted()/batchSummary.getTotalTransactions())*100;
+                    long percentage = (batchSummary.getSuccessful()/batchSummary.getTotal())*100;
```
Comment (fynmanoj) [inline src/main/java/org/mifos/processor/bulk/camel/routes/BatchStatusRoute.java:68]: why this is long? are we ignoring the decimal part ?  looks like this division is also an integer division. Is this expected behaviour?

File: src/main/java/org/mifos/processor/bulk/schema/BatchDTO.java [hunk 2]
```diff
@@ -5,73 +5,97 @@
 
 public class BatchDTO {
 
-    private String batchId;
+    private String batch_id;
```
Comment (fynmanoj) [inline src/main/java/org/mifos/processor/bulk/schema/BatchDTO.java:8]: why do we have the same file with a different naming convention for variables? some in camel case and some in snake case?

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/main/java/org/mifos/processor/bulk/camel/routes/BatchStatusRoute.java (hunk 1 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/processor/bulk/camel/routes/BatchStatusRoute.java:68]: We don't need a decimal value, its just a percentage threshold check.

File: src/main/java/org/mifos/processor/bulk/schema/BatchDTO.java (hunk 2 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/processor/bulk/schema/BatchDTO.java:8]: everything is followed as per the existing code.

Comment (fynmanoj) [reply src/main/java/org/mifos/processor/bulk/schema/BatchDTO.java:8]: New code should be better than existing code

**Pull request merged**